redfish-core/lib/message_registries.hpp

```diff
@@ -47,11 +48,11 @@ inline void handleMessageRegistryFileCollectionGet(
     asyncResp->res.jsonValue["Name"] = "MessageRegistryFile Collection";
     asyncResp->res.jsonValue["Description"] =
         "Collection of MessageRegistryFiles";
-    asyncResp->res.jsonValue["Members@odata.count"] = 4;
+    asyncResp->res.jsonValue["Members@odata.count"] = 5;
```

Comment:
Just wondering whether we can use size of Members' collection instead of hard-coding the numbers?
Maybe move this line right after line#61 and use the size of members variable?